### ISSUE TITLE
Remove Bootstrap dependency

### ISSUE THREAD

**Issue by chrishawn** [pleerock/ng2-tooltip#18]
Why don't you make this self contained... insert your own css with it, instead of adding the Bootstrap dependency.

**Comment by pleerock** [comment]
yes it is possible to do it, however Im not good with css. I need some help there

**Comment by AdamSGit** [comment]
Why dont you just take the tooltip part of the bootstrap css, (that you can find as a sass partial) and include it in your package ?

I would have done it for you, but i'm not sure how to include css in an angular module.
